CODE_REVIEW.md

```diff
@@ -0,0 +1,18 @@
+CODE IMPROVEMENTS
+1. In the component "book-search", subscribe is used but the subscription is no where being caught and destroyed. (FIXED)
+- Failing to unsubscribe may lead to memory leaks and to solve this, either we have to unsubscribe the subscription or can make use of an async pipe.
+- The subscription started in ngOnInit of BookSearchCompoent is never unsubscribed - possible memory leak
+Observable used for this.store.select(getAllBooks) is not unsubscribed in book-search.component.ts file. This may lead to memory leak and performance issue.
+-	Added async pipe in 'book-search.component.html' file line#33 to avoid memory leak. Async pipe will automatically unsubscribe the observable once component is destroyed
+
+2.	Custom method formateDate() is being used for date format in book-search.component.html at line#45. (FIXED)
+- We can use pipe instead to convert date into required format. This will help in performance improvement since function will be triggered everytime on change detection and wherein having pipe would evaluate the expression only once.
+
+ACCESSIBILITY FIXES
```

Comment:
Can you please list down separately what you found manually and what you found with Lighthouse?